src/os/bluestore/bluestore_types.h

```diff
@@ -156,7 +157,7 @@ WRITE_CLASS_DENC(bluestore_pextent_t)
 
 ostream& operator<<(ostream& out, const bluestore_pextent_t& o);
 
-typedef mempool::bluestore_meta_other::vector<bluestore_pextent_t> PExtentVector;
+typedef boost::container::small_vector<bluestore_pextent_t, 1> PExtentVector;
```

Comment:
Yup, that's entirely true. I think it's worth to introduces helpers in `mempool.h` for couple useful containers from Boost. I will take of that.